### PR TITLE
[MX-50] -  Clean up from Artwork rebuild release

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/MX-50
- Removes old Artwork View Controllers

### CODE REVIEW

File: Artsy/View_Controllers/Fair/ARProfileViewController.m [hunk 1]
```diff
@@ -82,7 +82,7 @@ - (void)loadProfile
                 NSString * fairID = ((Fair *) profile.profileOwner).fairID;
                 ARFairComponentViewController *viewController = [[ARFairComponentViewController alloc] initWithFairID:fairID];
                 [self showViewController:viewController];
-            } else if ([profile.profileOwner isKindOfClass:[Partner class]] && [AROptions boolForOption:AROptionsNewPartnerPage]) {
+            } else if ([profile.profileOwner isKindOfClass:[Partner class]]) {
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Fair/ARProfileViewController.m]: Do we want to replace this with a new Echo flag?

File: CHANGELOG.yml [hunk 2]
```diff
@@ -6,6 +6,14 @@ upcoming:
...
     - Adds basic UI infrastructure for CCPA UI - ash
     - Upgrates to Xcode 11 - ash
+    - Switched CocoaPods to use the CDN master spec repo - alloy
+    - Add a new flag for Price Transparency to Echo - yuki24
+    - Increases minimum deploy version to iOS 12 - ash
+    - Adds option to show a collection view controller - ash
+    - Upgrades React Native to 0.61.4 - ash & david & kieran
+    - Add back-button-hiding infra for emission - ash & david & kieran
+    - Puts new partner page behind an Echo flag - ash
+    - Removes references to legacy artwork view - kieran
```
Comment (ashfurrow) [inline CHANGELOG.yml]: Looks like a merge conflict may have led to duplicate entries here.

File: Podfile [hunk 3]
```diff
@@ -114,7 +114,6 @@ target 'Artsy' do
   pod 'SSFadingScrollView', git: 'https://github.com/alloy/SSFadingScrollView.git', branch: 'add-axial-support'
 
   # Core owned by Artsy
-  pod 'ARTiledImageView', git: 'https://github.com/dblock/ARTiledImageView'
```
Comment (ashfurrow) [inline Podfile]: 🙌

File: CHANGELOG.yml (hunk 2 above)
Comment (kierangillen) [reply CHANGELOG.yml]: 👍

**Pull request merged**